pygmt/src/magnetic_rose.py

```diff
@@ -0,0 +1,119 @@
+"""
+magnetic_rose - Add a map magnetic rose.
+"""
+
+from collections.abc import Sequence
+from typing import Literal
+
+from pygmt.alias import Alias, AliasSystem
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import build_arg_list
+from pygmt.params import Box, Position
+
+
+def magnetic_rose(  # noqa: PLR0913
+    self,
+    position: Position | None = None,
+    width: float | str | None = None,
+    labels: Sequence[str] | bool = False,
+    outer_pen: str | bool = False,
+    inner_pen: str | bool = False,
+    declination: float | None = None,
+    declination_label: str | None = None,
+    intervals: Sequence[float] | None = None,
+    box: Box | bool = False,
+    perspective: str | bool = False,
+    verbose: Literal["quiet", "error", "warning", "timing", "info", "compat", "debug"]
+    | bool = False,
+    transparency: float | None = None,
+):
+    """
+    Add a magnetic rose to the map.
+
+    Parameters
```

Comment:
```suggestion
    Add a magnetic rose to the map.

    $aliases
       - V = verbose
       - p = perspective
       - t = transparency
       
    Parameters
```

Reply:
I'm debating whether to add aliases in wrappers like this one, given that they are new wrappers not mapped to a single GMT module and therefore lack a one-to-one mapping.